apps/passport-server/src/services/generic-issuance/pipelines/CSVPipeline/makeGenericPODPCD.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  CSVPipelineMatchConfig,
+  CSVPipelinePODEntryOptions
+} from "@pcd/passport-interface";
+import { ArgumentTypeName, SerializedPCD } from "@pcd/pcd-types";
+import { PODEntries } from "@pcd/pod";
+import { PODPCD, PODPCDPackage } from "@pcd/pod-pcd";
+import { v5 as uuidv5 } from "uuid";
+import { traced } from "../../../telemetryService";
+
+export async function makeGenericPODPCD(
+  inputRow: Record<string, string>,
+  eddsaPrivateKey: string,
+  requesterEmail: string | undefined,
+  requesterSemaphoreId: string | undefined,
+  pipelineId: string,
+  issueToUnmatchedEmail: boolean | undefined,
+  output: CSVPipelinePODEntryOptions,
+  matchConfig?: CSVPipelineMatchConfig
+): Promise<SerializedPCD<PODPCD> | undefined> {
+  return traced("", "makeGenericPODPCD", async () => {
+    if (!requesterEmail || !requesterSemaphoreId) {
```

Comment:
Shouldn't issueToUnmatchedEmail be involved here?
I'm actually not clear on the intended semantics of this field.
Is it for cases where the user is required to have an email address, but it doesn't match the data?  Or is it for cases where no email matching is necessary, and all users get the same data?  Are there ways to represent both?

Reply:
This is mostly due to the fact that I'm trying to implement something on top of the existing implementation of `CSVPipeline`, which assumes that `requesterEmail` and `requesterSemaphoreId` might be undefined. This doesn't make a lot of sense to me, but I don't want to change code that's unrelated to the feature I'm trying to implement. This has nothing to do with `issueToUnmatchedEmail`, which needs there to be a `requesterEmail` in order to issue anything at all - it just doesn't have to match any of the emails in the `email` column of the CSV file.

Reply:
I could imagine supporting feeds delivered to users without any authentication required, in which case email and semaphore could be undefined.  This would only work for PODs which don't have an owner (i.e. no entries filled in with the subscriber's email or semaphore), which could be valid if you're just handing out a fixed collectible, or a public feed of podcasts or articles, etc.

I don't know if that's actually a feature supported by existing CSV pipelines for message PCDs.  It wouldn't make sense for ticket PCDs, which always have an owner.  I think supporting it for PODs would be useful, but not a high-priority feature.